UnityProject/Assets/Scripts/Items/Cards/Emag.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Mirror;
+
+/// <summary>
+/// Emag charges handler
+/// </summary>
+public class Emag : NetworkBehaviour
+{
+	private SpriteHandler spriteHandler;
+
+    [Tooltip("Number of charges emags start with")]
+    [SerializeField]
+    public int startCharges = 3;
```

Comment:
https://unitystation.github.io/unitystation/development/SyncVar-Best-Practices-for-Easy-Networking/ 
SyncVar so, client knows how many charges it got left, might be handled by validation, if it reduces a validation message then good enough

Reply:
sorry I mean charges* not startCharges